lib/check-links.js

```diff
@@ -1,11 +1,16 @@
 import pMap from 'p-map'
 import pMemoize from 'p-memoize'
+import ExpiryMap from 'expiry-map'
 
 import { checkLink } from './check-link.js'
 
-const isUrlAlive = pMemoize(checkLink, {
-  maxAge: 60 * 1000
-})
+/** @type {import('expiry-map')<string, import('./check-link.js').LivenessResult>} */
+const cache = new ExpiryMap(60 * 1000)
+const isUrlAlive = pMemoize(checkLink, { cache })
```

Comment:
the `maxAge` option was removed in version 6 https://github.com/sindresorhus/p-memoize/releases/tag/v6.0.0
This is the recommended alternative

Reply:
great call